c++/src/kj/memory.h

```diff
@@ -22,7 +22,13 @@
 #pragma once
 
 #include "common.h"
-#ifdef KJ_DEBUG
+
+#if defined(KJ_DEBUG) && !defined(KJ_ASSERT_PTR_COUNTERS)
+#define KJ_ASSERT_PTR_COUNTERS
```

Comment:
This is not new in your change and I'm not asking you to fix it in your PR, but:

This really ought to be defined with the value `1`, and the `#ifdef KJ_ASSERT_PTR_COUNTERS` should be `#if KJ_ASSERT_PTR_COUNTERS`, so that someone can explicitly define it to `0` if they want to disable this while still building in debug mode. Most KJ feature macros work this way (although `KJ_DEBUG` in an exception, for no good reason).

Reply:
fyi @mikea